nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestGetFileResource.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.standard;
+
+import org.apache.nifi.flowfile.attributes.CoreAttributes;
+import org.apache.nifi.util.MockFlowFile;
+import org.apache.nifi.util.TestRunner;
+import org.apache.nifi.util.TestRunners;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Unit tests for the GetFileResource processor.
+ */
+public class TestGetFileResource {
+
+    @Test
+    public void testGetFileResource() throws IOException {
```

Comment:
What do you think about splitting up this test into multiple test cases?
I know there are a lot of tests in the NiFi codebase in this style but I'm my opinion, they document intent rather poorly. 

From what I've gathered you basically have the following test cases:
- isInvalidWhenFileResourceIsNotSet
- isValidWhenFileResourceIsSet
- createsAFlowFileWithTheFileResourceAsItsContent
- supportsSpecifyingMimeType
- supportsSpecifyingAdditionalAttributes